exercises/word-count/example.scm

```diff
@@ -0,0 +1,27 @@
+(define-module (word-count)
+  #:use-module (srfi srfi-1)
+  #:use-module (ice-9 regex)
+  #:export (countwords))
+
+
+(define (countword word counters)
```

Comment:
We could instead do something like the following, but this is good enough as is.

```scheme
(define (countword word counters)
  (let ((newcount (1+ (or (assoc-ref counters word) 0))))
    (assoc-set! counters word newcount)))
```